src/openstudio_lib/OSDropZone.cpp

```diff
@@ -929,4 +913,43 @@ boost::optional<model::ModelObject> OSDropZone2::updateGetterResult() {
   return m_getterResult;
 }
 
+void OSDropZone2::updateOtherModelObjects() {
```

Comment:
Hey Julien, I agree.  I added `otherObjects` to the `DropZoneConcept` when I refactored the grid view code.  The `otherObjects` function tells the drop zone to watch some other object (in this case the thermostat or humidistat) for changes and refresh the drop zone if that other object changes in addition to changes in the main object (in this case the zone).  The problem here was that the return value of `otherObjects` changes when the thermostat/humidistat are removed/added.  Previously, drop zone only called `otherObjects` at bind time (which is why I created thermostat/humidistat at bind time in my refactor).  Now, the drop zone calls `otherObjects` at each refresh.  I made the `otherObjects` function take a `const T*` because if you change the main object inside the `otherObjects` function you will get an infinite loop of calling `refresh` then `otherObjects`.